tests/pytype_test.py

```diff
@@ -122,11 +126,10 @@ def pytype_test(args):
             if f in pytype_run:
                 test_run = BinaryRun(
                     ['pytype',
-                     '--typeshed-location=%s' % os.getcwd(),
                      '--module-name=%s' % _get_module_name(f),
-                     '--convert-to-pickle=%s' % os.devnull,
-                     f],
-                    dry_run=args.dry_run)
+                     '--parse-pyi', f],
```

Comment:
I'd keep `f` on a line by itself (like it was -- you made it "hide" between a much longer flag).

Reply:
Done.